ubuntu_install_requirements.sh

```diff
@@ -18,36 +18,39 @@ sudo apt -y update
 # Install required packages
 
 sudo apt -y install \
+  python3-pip \
+  python-pip \
+  python-setuptools \
```

Comment:
Can you please put a comment why we need this ? (ansible using default python to run on the local machine)

Reply:
Done